editoast/src/models/fixtures.rs

```diff
@@ -308,3 +310,27 @@ pub async fn create_work_schedule_group(conn: &mut DbConnection) -> WorkSchedule
         .await
         .expect("Failed to create empty work schedule group")
 }
+
+pub struct WorkSchedulesFixtureSet {
+    pub work_schedule_group: WorkScheduleGroup,
+    pub work_schedules: Vec<WorkSchedule>,
+}
+
+pub async fn create_work_schedules_fixture_set(
+    conn: &mut DbConnection,
+    work_schedules: Vec<WorkScheduleItemForm>,
+) -> WorkSchedulesFixtureSet {
```

Comment:
We used to need `XFixtureSet` because we had to wrap that into `TestFixture<T>`, which is gone now. You can just return a tuple.

```suggestion
pub async fn create_work_schedules_fixture_set(
    conn: &mut DbConnection,
    work_schedules: Vec<WorkScheduleItemForm>,
) -> (WorkScheduleGroup, Vec<WorkSchedule>) {
```

Reply:
done !